packages/medusa/src/api/routes/admin/returns/cancel-return.ts

```diff
@@ -42,3 +44,7 @@ export default async (req, res) => {
 
   res.status(200).json({ order })
 }
+
+export type AdminPostReturnsReturnCancel = {
```

Comment:
(We should probably modify this so that this doesn't respond with an order but that is another discussion)

All responses should be exported from `admin/returns/index.ts` so please move it there; given that this is an odd one out I would suggest something like `AdminReturnsCancelRes`